contrib/socat/template.py

```diff
@@ -11,12 +11,12 @@
     "openssl-devel",
     "readline-devel",
 ]
+checkdepends = ["bash", "iproute2", "procps"]
+depends = ["bash"]
```

Comment:
socat itself doesn't need bash, the only thing that does are the *.sh scripts that i have never seen anyone use (they look more like examples than anything)

Reply:
moved to usr/share/examples